### PR TITLE
append TOTAL CPU time Pie chart diagram

### PR DESCRIPTION
**Pull request by Slach**
Signed-off-by: Eugene Klimov <eklimov@altinity.com>

### CODE REVIEW

File: deploy/grafana/grafana-with-grafana-operator/grafana-dashboard-operator-cr-template.yaml [hunk 1]
```diff
@@ -13,5 +13,7 @@ spec:
   # TODO remove this plugin definition after resolve https://github.com/integr8ly/grafana-operator/issues/155
   plugins:
     - name: "vertamedia-clickhouse-datasource"
-      # update to 2.0.4 when resolve https://github.com/grafana/grafana-plugin-repository/pull/690
+      # update to 2.1.0 when next grafana-operator release
       version: "1.9.5"
+    - name: "grafana-piechart-panel"
```
Comment (Slach) [inline deploy/grafana/grafana-with-grafana-operator/grafana-dashboard-operator-cr-template.yaml:18]: @alex-zaitsev after these changes will merge to `master` we need to create separate separate PR to merge to `0.12.0` branch cause ACM look to 0.12.0 `grafana-dashboard-operator-cr-template.yaml`

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json [hunk 2]
```diff
@@ -3663,7 +3663,7 @@
       "timeFrom": null,
       "timeRegions": [],
       "timeShift": null,
-      "title": "CPU Time microseconds per seconds",
+      "title": "ClickHouse CPU Time per seconds",
```
Comment (alex-zaitsev) [inline grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3666]: should be "CPU Time per second"

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json [hunk 3]
```diff
@@ -3844,7 +3844,7 @@
       ],
       "timeFrom": null,
       "timeShift": null,
-      "title": "Total ClickHouse CPU Time microseconds",
+      "title": "ClickHouse CPU Time total",
```
Comment (alex-zaitsev) [inline grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3847]: CPU Time Total
